packages/FSharp.Formatting/FSharp.Formatting.fsx

```diff
@@ -1,76 +0,0 @@
-#nowarn "211"
```

Comment:
I think we might actually need this file (for a release or something?), though I do not remember the details. /cc @matthid who probably knows more.... I think there was some intricate reason why the file had to be in this particular folder, even though it's not a normal place where we would put it.